pages/privacy.js

```diff
@@ -0,0 +1,254 @@
+import { Text, ListItem } from '@chakra-ui/core'
+import { UnorderedList } from '@chakra-ui/react'
+
+import PageWrapper from '../components/common/pageWrapper'
+import Section from '../components/common/section'
+import Card from '../components/common/card'
+import UnderlinedHeading from '../components/common/underlinedHeading'
+import TextLink from '../components/common/textLink'
+import FAQHeading from '../components/common/faqHeading'
+import Divider from '../components/common/divider'
+
+const PrivacyTextBlock = ({ children, ...props }) => (
+  <Text marginBottom='1.5rem' {...props}>{children}</Text>
+)
+
+const Privacy = () => (
+  <PageWrapper title='Privacy Policy'>
+    <Section
+      display='flex'
+      justifyItems='center'
+      flexDirection='column'
+      alignItems='center'
+      padding={{ base: '3rem 1.5rem', lg: '3rem 7.5rem 6rem' }}
+      backgroundColor='lightRock'
+    >
+      <UnderlinedHeading as='h1' text='Privacy Policy' align='center' />
+      <Card shadowSz='lg' maxW='60rem'>
+        <FAQHeading>TLDR</FAQHeading>
+        <PrivacyTextBlock>
+          We know privacy policies can be riddled with heavy language and legalese that’s difficult to parse. Our TLDR is that we will not share your personal information no matter what.
+        </PrivacyTextBlock>
+        <PrivacyTextBlock fontWeight='500'>
+          The only information we have is your email address (used for authentication), your GitHub Id (used to see if you’re a GitHub admin for an organization you’re trying to donate on behalf of) and the OSS packages you’ve installed (used to distribute your donation / ad revenue).
+        </PrivacyTextBlock>
+        <PrivacyTextBlock fontWeight='500'>
+          That's it
+        </PrivacyTextBlock>
+        <PrivacyTextBlock>
+          Please accept the following privacy policy as the official language of what’s stated above.
+        </PrivacyTextBlock>
+        <Divider />
+        <FAQHeading>Privacy Policy</FAQHeading>
+        <PrivacyTextBlock>
+          Our Privacy Policy was posted on 20 November 2020 and last updated on 20 November 2020. It governs the privacy terms of our website, located at https://flossbank.com, and the tools we provide you (the "Website" or the "Service").
```

Comment:
```suggestion
          Our Privacy Policy was posted on 20 November 2020 and last updated on 20 November 2020. It governs the privacy terms of our website, located at https://flossbank.com, and the tools we provide (the "Website" or the "Service").
```

Taking off the "you" since the parenthetical becomes unclear with it present.